package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@maestro-org/typescript-sdk",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
Thanks @SIDANWhatever for your important contributions :pray:

Regarding chosen version change, I am not sure if this is appropriate as breaking changes should lead to increase in major when major is >= 1. https://semver.org/.

Reply:
@sourabhxyz My preference is that we use the "minor" for breaking changes and the "patch" for everything else. The major version should match our API, which is currently V1.